packages/devtools_app/lib/src/extensions/embedded/_controller_web.dart

```diff
@@ -80,7 +80,7 @@ class EmbeddedExtensionControllerImpl extends EmbeddedExtensionController
     );
     _initialized = true;
 
-    _extensionIFrame = createIFrameElement()
+    _extensionIFrame = HTMLIFrameElement()
```

Comment:
is this equivalent to what `createIFrameElement` used to return? If so, LGTM.

Reply:
Yup! `createIFrameElement()` is now deprecated in favor of the `HTMLIFrameElement()` constructor.